### ISSUE TITLE
The mysterious case of the duck skin

### ISSUE THREAD

**Issue by luanpotter** [luanpotter/bgug#6]
For some reason there is a duck skin on the skins json:

```
    {
      "name": "Duck",
      "file": "duck.png",
      "cost": 200
    },
```

However it appears that there is and there has never been any PNG/ase files regarding that skin.

**Comment by luanpotter** [comment]
Quack?

**Comment by luanpotter** [comment]
![image](https://user-images.githubusercontent.com/882703/79613330-51a9fd00-80cc-11ea-809c-62cfa5cb7da1.png)

**Comment by luanpotter** [comment]
Closed by #8
